src/scenes/Projects/List/ProjectList.tsx

```diff
@@ -25,12 +25,16 @@ const useStyles = makeStyles(({ spacing }) => ({
   projectItem: {
     marginBottom: spacing(2),
   },
+  projectTotal: {
+    margin: spacing(2, 2),
+  },
 }));
 
 export const ProjectList: FC = () => {
   const formatNumber = useNumberFormatter();
   const sort = useSort<Project>();
   const [filters, setFilters] = useProjectFilters();
+  const [activeTab, setActiveTab] = useState('mine');
```

Comment:
This should be stored in the `useProjectFilters` state. This way page reloads & navigation bring users back to the same tab. We also define the filters so they can be passed straight to API filters.
So we need to add a `pinned` property. Maybe it and `mine` can share a `tab` enum query param?

The mine toggle should also be removed from `<ProjectFilterOptions />`